skyportal/models.py

```diff
@@ -188,6 +203,13 @@ def desi_dr8_url(self):
                             secondary='join(Group, group_sources).join(group_users)',
                             primaryjoin='group_users.c.user_id == users.c.id')
 
+GroupSource.saved_as_source_by_id = sa.Column(sa.ForeignKey("users.id"), nullable=True)
+GroupSource.saved_as_source_by = relationship(
```

Comment:
Curious how you feel about the naming of all the `join_model` classes.  It feels more intuitive to me to have a plural, to indicate the one to many relationship: `GroupSources`, `StreamGroups`, etc.  Thoughts?

Reply:
This is in keeping with our model naming convention, though. We have `Comment`, `Source`, `Group`, `User`, etc.